ui/pages/confirmations/components/signature-request-original/signature-request-original.component.js

```diff
@@ -313,10 +311,6 @@ export default class SignatureRequestOriginal extends Component {
           history.push(mostRecentOverviewPage);
         }}
         onSubmit={async () => {
-          if (txData.type === MESSAGE_TYPE.ETH_SIGN) {
-            return this.setState({ showSignatureRequestWarning: true });
```

Comment:
Can't we delete the state and component for `showSignatureRequestWarning` too?

Reply:
https://github.com/MetaMask/metamask-extension/pull/24756/commits/343ee961208c68f42397d6591f73f14d2c74891e